padertorch/ops/stft_module.py

```diff
@@ -0,0 +1,270 @@
+import numpy as np
```

Comment:
Why is this file called `stft_module`? This doesn't match the other files in `padertorch/ops`. As long as we only have an `STFT` class, there's no name clash

Reply:
I used this name to allow the addition of a stft function. But this is of course open for discussion. If most of you do not think we need the stft function or prefer an other name we can change it.

Reply:
I am also in favor to keep the name `stft` free, so that we can implement the function as alternative to the class.

Reply:
okay so should we keep the name or change it to something like transform?

Reply:
I also think that we should keep `stft` free for a functional implementation. But I also don't like the suffix `_module` because it is not used anywhere else (in `padertorch`). 

I just noticed that `einsum` module contains a function `einsum`. So, I think this should be similar in all sub-modules of `pt.ops`, so it should either be `stft` with everything exported to `__init__` (as in `einsum`), or all other modules have to be renamed as well.

Reply:
changed this to _stft.py